cmd/cmount/fs.go

```diff
@@ -550,7 +550,22 @@ func (fsys *FS) Setxattr(path string, name string, value []byte, flags int) (err
 
 // Getxattr gets extended attributes.
 func (fsys *FS) Getxattr(path string, name string) (errc int, value []byte) {
-	return -fuse.ENOSYS, nil
+	defer log.Trace(path, "name=%q", name)("errc=%d", &errc)
+
+	node, handle, errc := fsys.getNode(path, fhUnset)
```

Comment:
Do you think it's worth grabbing the handle here considering we're passing `fhUnset` into `getNode`? From what I can tell, `getNode` will always return a `nil` handle in this case, so adding the code to cover the case where it doesn't would only be future-proofing if that behavior ever changes.

Reply:
If we don't have an `fh` then probably best to explicitly call this instead.

    node, errc := fsys.lookupNode(path)

Reply:
Done!